File: openapi/code_samples/csharp/api_company_sector_{caen}_distribution.cs
using Microsoft.AspNetCore.Mvc;
using Swashbuckle.AspNetCore.Annotations;

[ApiController]
[Route("api/[controller]")]
public class CompanyController : ControllerBase
{
    [HttpGet]
    [Route("sector-distribution/{caen}")]
    [SwaggerOperation(Summary = "Retrieves the company distribution on a given sector CAEN.")]
    [ProducesResponseType(typeof(SectorDistributionByFiscalValueView[]), 200)]
    [ProducesResponseType(typeof(Fault), 400)]
    public IActionResult GetSectorDistributionByCaen([FromRoute] string caen)
    {
        // Your implementation here
        return Ok();
    }
}

File: openapi/code_samples/python/api_company_sector_{caen}_distribution.py
from flask import Flask, request, jsonify

app = Flask(__name__)

@app.route('/api/company/sector-distribution/<string:caen>', methods=['GET'])
def get_sector_distribution_by_caen(caen):
    # Your implementation here
    return jsonify({}), 200

if __name__ == '__main__':
    app.run()
